src/test/java/org/ahocorasick/trie/PayloadTrieTest.java

```diff
@@ -461,6 +462,24 @@ public void testWholeWords() {
         Collection<PayloadEmit<String>> result2 = new LinkedList<>();
         trie.parseText("foobar", result2::add);
 
+        assertTrue(result1.isEmpty());
+        assertEquals(result1, result2);
+    }
+
+    // @see https://github.com/robert-bor/aho-corasick/issues/85
+    @Test
+    public void wholeWordsWhiteSpaceSeparated() {
+        PayloadTrie<String> trie = PayloadTrie.<String>builder().addKeyword("foo", "bar").onlyWholeWordsWhiteSpaceSeparated().build();
+
+        // access via PayloadTrie.parseText(CharSequence)
+        Collection<PayloadEmit<String>> result1 = trie.parseText("foo#bar");
+
```

Comment:
Style nit: Prefer to avoid double-blank lines.

Reply:
👍